app/controllers/items_controller.rb

```diff
@@ -15,6 +10,11 @@ def create
     end
   end
 
+  def index
+    @item = Item.new
+    @category = Category.new
```

Comment:
I am actually using it to render the category form. Do you think I should just keep Category.new there and skip the instance variable?